LayoutTests/scrollbars/scrolling-backward-by-page-accounting-bottom-fixed-elements-on-keyboard-spacebar.html

```diff
@@ -1,3 +1,4 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true EventHandlerDrivenSmoothKeyboardScrollingEnabled=true ] -->
```

Comment:
This *feels* wrong; the test will test the feature when we turn it on, but I don't see why we need to do this? (The test wasn't about smooth keyboard scrolling when written, so it shouldn't force it on).